hunter/analysis.py

```diff
@@ -0,0 +1,241 @@
+from dataclasses import dataclass
```

Comment:
Hm, instead of copying the hunter code, could we just not install hunter via pip?